exampleCourse/questions/element/truthTable/info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "uuid": "5150658a-7ec7-4a96-912a-353da738bef0",
+    "title": "Training: Truth Table",
+    "topic": "Training",
+    "tags": ["grantleech", "formative", "easy", "blank", "python", "alpha", "csulb"],
```

Comment:
The alpha, blank, and formative tags are all used to indicate what type a question is or what stage of the review process it's in for csulb's questions. I'll go ahead and remove these since they don't serve much of a purpose of that context.